### PR TITLE
WinUI-3340 : Prepare UG document content for Placeholder value and IsClear Enabled support in winUI.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared UG document content for PlaceholderValue and IsClearEnabled support.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -463,6 +463,8 @@
            <li><a href="/winui/Rating/Precision">Precision</a></li>
            <li><a href="/winui/Rating/ToolTip">ToolTip</a></li>
            <li><a href="/winui/Rating/Customization">Customization</a></li>
+           <li><a href="/winui/Rating/placeholder">Placeholder</a></li>
+           <li><a href="/winui/Rating/clear-enabled">ClearEnabled</a></li>
```
Comment (Manivannan-E) [inline winui-toc.html:467]: @Priyanka-Vijayasankar 
placeholder-value => Placeholder Value
clear-rating-value => Clear rating value

Comment (Priyanka-Vijayasankar) [reply winui-toc.html]: Updated the above changes.

File: winui/Rating/Clear-rating.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to enable the clear enable feature in the Rating control the user can remove their rating or not.
+platform: WinUI
```
Comment (Manivannan-E) [inline winui/Rating/Clear-rating.md:5]: @Priyanka-Vijayasankar Learn all about how to clear the rating value in the WinUI Rating (SfRating) control.

File: winui/Rating/Clear-rating.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+control: SfRating
+documentation: ug
+---
+
+# Clear rating value in WinUI Rating
+
+The Rating control provides value support to determine whether a user can remove their rating or not, by using the `IsClearEnabled` property.
+
+`Note`- It supports Precision `full` value only.
+
```
Comment (Manivannan-E) [inline winui/Rating/Clear-rating.md:15]: @Priyanka-Vijayasankar It supports precision mode full only.

File: winui/Rating/Clear-rating.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Clear rating value in WinUI Rating
+
+The Rating control provides value support to determine whether a user can remove their rating or not, by using the `IsClearEnabled` property.
+
+`Note`- It supports Precision `full` value only.
```
Comment (Manivannan-E) [inline winui/Rating/Clear-rating.md:14]: @Priyanka-Vijayasankar Check and change the note.

File: winui/Rating/Placeholder-value.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+description: Learn all about the placeholder feature in the Rating control to show the average rating value before the user provide their own rating.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Placeholder value in WinUI Rating
+
+The Rating control provides the average rating value before the user provide or click any other actions by using the `PlaceholderValue` property.
+
```
Comment (Manivannan-E) [inline winui/Rating/Placeholder-value.md:13]: @Priyanka-Vijayasankar By using the 'PlaceholderValue' property, the Rating control can display the average rating value before the user provides their own rating by click or other action.

File: winui/Rating/Clear-rating.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+title: Clear rating value in WinUI Rating control | Syncfusion
+description: Learn all about how to enable the clear enable feature in the Rating control the user can remove their rating or not.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# Clear rating value in WinUI Rating
+
+The Rating control provides value support to determine whether a user can remove their rating or not, by using the `IsClearEnabled` property.
```
Comment (Manivannan-E) [inline winui/Rating/Clear-rating.md:12]: @Priyanka-Vijayasankar The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true.

File: winui/Rating/Clear-rating.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:5]: Updated the above changes.

File: winui/Rating/Clear-rating.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:15]: Updated the above changes.

File: winui/Rating/Clear-rating.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md]: Check and Updated the above changes.

File: winui/Rating/Placeholder-value.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:13]: Updated the above changes.

File: winui/Rating/Clear-rating.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md]: Updated the above changes.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Clear-rating.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:2]: Layout: Post

File: winui/Rating/Clear-rating.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+title: Clear rating value in WinUI Rating control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:3]: Title: Clear rating value in WinUI Rating control | Syncfusion

File: winui/Rating/Clear-rating.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+description: Learn all about how to clear the rating value in the WinUI Rating (SfRating) control.
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:4]: Description: Learn how to clear the rating value in the WinUI Rating (SfRating) control.

File: winui/Rating/Clear-rating.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:5]: Platform: WinUI

File: winui/Rating/Clear-rating.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+control: SfRating
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:6]: Control: SfRating

File: winui/Rating/Clear-rating.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:7]: Documentation: UG

File: winui/Rating/Clear-rating.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+# Clear rating value in WinUI Rating
+
+The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true. The default value of the `IsClearEnabled` property is `true`.
```
Comment (Chrispineagunja) [inline winui/Rating/Clear-rating.md:12]: The `IsClearEnabled` property of the Rating control allows you to clear the provided rating value when the value is true. The default value of the `IsClearEnabled` property is `true.`

File: winui/Rating/Placeholder-value.md [hunk 14]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:2]: Layout: Post

File: winui/Rating/Placeholder-value.md [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+title: Placeholder value in WinUI Rating control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:3]: Title: Placeholder value in WinUI Rating control | Syncfusion

File: winui/Rating/Placeholder-value.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+description: Learn all about the placeholder feature in the Rating control to show the average rating value before the user provide their own rating.
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:4]: Description: Learn about the placeholder feature in the Rating control to show the average rating value before the user provides their rating.

File: winui/Rating/Placeholder-value.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
...
+platform: WinUI
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:5]: Platform: WinUI

File: winui/Rating/Placeholder-value.md [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+control: SfRating
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:6]: Control: SfRating

File: winui/Rating/Placeholder-value.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:7]: Documentation: UG

File: winui/Rating/Placeholder-value.md [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+# Placeholder value in WinUI Rating
+
+By using the `PlaceholderValue` property, the Rating control can display the average rating value before the user provides their own rating by click or any other action.
```
Comment (Chrispineagunja) [inline winui/Rating/Placeholder-value.md:12]: By using the `PlaceholderValue` property, the Rating control display the average rating value before the user provides their rating by click or any other action.

File: winui/Rating/Clear-rating.md (hunk 9 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:4]: Updated the above mentioned content changes.

File: winui/Rating/Clear-rating.md (hunk 13 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:12]: Updated the above mentioned changes.

File: winui/Rating/Placeholder-value.md (hunk 16 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:4]: Updated the above mentioned changes.

File: winui/Rating/Placeholder-value.md (hunk 20 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:12]: Updated the above mentioned changes.

**Comment by Priyanka-Vijayasankar** [comment]
We have updated the mentioned changes in the below PR,
[PR Link](https://github.com/syncfusion-content/winui-docs/pull/555)

File: winui/Rating/Clear-rating.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:2]: We have follow the small letter for all other controls.

File: winui/Rating/Clear-rating.md (hunk 8 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:3]: We have follow the small letter for all other controls.

File: winui/Rating/Clear-rating.md (hunk 10 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:5]: We have follow the small letter for all other controls.

File: winui/Rating/Clear-rating.md (hunk 11 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:6]: We have follow the small letter for all other controls.

File: winui/Rating/Clear-rating.md (hunk 12 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Clear-rating.md:7]: We have follow the small letter for all other controls.

File: winui/Rating/Placeholder-value.md (hunk 14 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:2]: We have follow the small letter for all other controls.

File: winui/Rating/Placeholder-value.md (hunk 15 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:3]: We have follow the small letter for all other controls.

File: winui/Rating/Placeholder-value.md (hunk 17 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:5]: We have follow the small letter for all other controls.

File: winui/Rating/Placeholder-value.md (hunk 18 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:6]: We have follow the small letter for all other controls.

File: winui/Rating/Placeholder-value.md (hunk 19 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Placeholder-value.md:7]: We have follow the small letter for all other controls.

**Pull request merged**